mobile/app.json

```diff
@@ -1,5 +1,15 @@
 {
   "expo": {
-    "sdkVersion": "23.0.0"
+   "name": "Mots",
+   "icon": "./icons/soldevelo.png",
+   "version": "1.0.0",
+   "slug": "soldevelo-mots-app",
+   "sdkVersion": "23.0.0",
+   "ios": {
+     "bundleIdentifier": "com.yourcompany.yourappname"
+   },
+   "android": {
+     "package": "com.yourcompany.yourappname"
```

Comment:
is this needed? if so let's use: org.grameenfoundation.mots, same with ios

Reply:
Yes, I believe it is, without that there were some building errors.